scrapers/processors/json.go

```
@@ -170,7 +170,7 @@ func (e Extract) Extract(inputs ...v1.ScrapeResult) ([]v1.ScrapeResult, error) {
 			}
 		default:
 			opts := oj.Options{OmitNil: true, Sort: true, UseTags: true}
-			parsedConfig, err = oj.ParseString(oj.JSON(v, opts))
+			parsedConfig, err = oj.ParseString(oj.JSON(v, &opts))
```

You have to pass `opts` as pointer else it will be silently ignored